packages/accordion/website/pages/_app.js

```diff
@@ -1,7 +1,7 @@
-import '../styles/globals.css'
+import "../styles/globals.css";
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return <Component {...pageProps} />;
 }
 
-export default MyApp
+export default MyApp;
```

Comment:
might be nice to move these prettier changes to their own PR

Reply:
I think if i dont include this [lint fix commit f2f941f](https://github.com/replayio/devtools/pull/7100/commits/f2f941f66f0991929e752b6198d5595217fb0eac), Lint / Trunk check GH action will fail for this PR :(

Reply:
Appreciate your help here! Sorry about our linting, it is a little wonky right now. If you can remove the lint fix commit and keep these changes isolated to the one file I'll get this merged. We can take a look at linting in a follow-up PR and should probably find a time to run it across the whole codebase anyways.

Reply:
@souporserious removed the lint fix commit. Please have a look 🙈